Flaky Junit test: EndToEndTxnWithScaleTest

**Problem description**
```
io.pravega.test.integration.endtoendtest.EndToEndTxnWithScaleTest > testTxnWithScale FAILED

    java.lang.AssertionError: expected:<txntest2> but was:<null>

        at org.junit.Assert.fail(Assert.java:88)

        at org.junit.Assert.failNotEquals(Assert.java:834)

        at org.junit.Assert.assertEquals(Assert.java:118)

        at org.junit.Assert.assertEquals(Assert.java:144)

        at io.pravega.test.integration.endtoendtest.EndToEndTxnWithScaleTest.testTxnWithScale(EndToEndTxnWithScaleTest.java:146)
```

**Problem location**
EndToEndTxnWithScaleTest

**Suggestions for an improvement**
Fix the sporadic junit test failure

--- a/test/integration/src/test/java/io/pravega/test/integration/endtoendtest/EndToEndTxnWithScaleTest.java
+++ b/test/integration/src/test/java/io/pravega/test/integration/endtoendtest/EndToEndTxnWithScaleTest.java
@@ -134,7 +134,8 @@ public class EndToEndTxnWithScaleTest {
         transaction.commit();
         @Cleanup
         ReaderGroupManager groupManager = new ReaderGroupManagerImpl("test", controller, clientFactory, connectionFactory);
-        groupManager.createReaderGroup("reader", ReaderGroupConfig.builder().build(), Collections.singleton("test"));
+        groupManager.createReaderGroup("reader", ReaderGroupConfig.builder().disableAutomaticCheckpoints().build(),
+                Collections.singleton("test"));
         @Cleanup
         EventStreamReader<String> reader = clientFactory.createReader("readerId", "reader", new JavaSerializer<>(),
                 ReaderConfig.builder().build());

